Source/engine.cpp

```diff
@@ -661,6 +734,11 @@ void CelClippedDrawLight(int sx, int sy, BYTE *pCelBuff, int nCel, int nWidth, i
 }
 
 /**
+ * @brief Same as CelClippedBlitLightTrans but with the option to skip parts of the top and bottom of the sprite

```

Comment:
Same as `CelClippedBlitLightTrans`... This brief is the documentation of `CelClippedBlitLightTrans`. So it's same as itself? :p Probably a copy-paste bug.